### ISSUE TITLE
Android: Unable to close web dialog from code

### ISSUE THREAD

**Issue by cmtanko** [tidev/titanium-web-dialog#13]
Instead of closing the web dialog manually, I want to close on certain callback or every timeout. How can I achieve this?

**Comment by cmtanko** [comment]
hey @m1ga , is it something that can be implemented with ease ? close() is only available for iOS and not android

**Comment by m1ga** [comment]
I don't think it is supported by the CustomTabs but you have to search for that in the CustomTabs documentation, I don't know that

**Comment by hansemannn** [comment]
That's unfortunately true, therefore closing this one for now.
